shared/editor/nodes/CodeFence.ts

```diff
@@ -134,12 +140,56 @@ export default class CodeFence extends Node {
           option.selected = node.attrs.language === value;
           select.appendChild(option);
         });
+        const codeClass = !node.attrs.diagram ? "editor-visible" : "";
+
+        const diagram = document.createElement("div");
+        if (node.attrs.language === "mermaidjs") {
+          const id = Math.round(Math.random() * 100000);
+          const diagramId = "mermaid-diagram-" + id;
+
+          diagram.classList.add("mermaid-diagram");
+          diagram.id = diagramId;
+          if (node.attrs.diagram) {
+            diagram.classList.add("diagram-visible");
+          }
+          diagram.setAttribute("contentEditable", "false");
+          try {
+            mermaid.mermaidAPI.render(diagramId, node.textContent, function (
+              svgCode: string
+            ) {
+              diagram.innerHTML = svgCode;
+            });
+          } catch (error) {
+            console.log(error);
+            const errorNode = document.getElementById("d" + diagramId);
+            if (errorNode) {
+              diagram.appendChild(errorNode);
+            }
+          }
+
+          const editButton = document.createElement("button");
+          if (node.attrs.diagram) {
+            editButton.innerText = "Show code";
```

Comment:
```suggestion
            editButton.innerText = this.options.dictionary.showCode;
```

These interface strings need adding to the dictionary object so that they can be translated.